climin/stops/stops.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, sec):
         self.start = time.time()
 
     def __call__(self, info):
-        return time.time() - self.start > self.sec
+        return getattr(info, 'time', time.time()) - self.start > self.sec
```

Comment:
But if `info` is a dictionary, it won't ever have that attribute, it should be `info.get('time', time.time())` to check for a *key* instead.